lib/config/presets/index.ts

```diff
@@ -45,6 +45,9 @@ const gitPresetRegex = regEx(
   /^(?<repo>~?[\w\-. /]+)(?::(?<presetName>[\w\-.+/]+))?(?:#(?<tag>[\w\-./]+?))?$/
 );
 
+// shallow mode preset to ignore regex
+const whitesourcePresetRegex = regEx(/^.*>(whitesource)\//);
```

Comment:
Is it written like this to capture both `local>` and `github>`? Maybe could be a tighter regex if so

Reply:
This regex matches presets that we want to ignore such as `github>whitesource/merge-confidence:beta`.
Now I'm Ignoring everything that is under `whitesource/`. If in the future we want to ignore other things they could be added here.
Are there other things that we want to currently ignore for resolving other than this and the internal presets ?